### PR TITLE
overview.md -> software/index.md

### PR DESCRIPTION
**Pull request by klml**
**Description**

It is important that each of these directories delivers meaningful content on its own. It makes sense for the user if he does not find the click-me-up button to simply delete something from the URL.

## Todo

* [x] redirect de/overview.html': 'de/software/',

### CODE REVIEW

File: .vitepress/config.mts [hunk 1]
```diff
@@ -90,6 +91,11 @@ export default defineConfig({
 
   },
 
+  rewrites: {
```
Comment (klml) [inline .vitepress/config.mts:94]: @FabianWilms I trid https://vitepress.dev/guide/routing#route-rewrites but this does not work

Comment (FabianWilms) [reply .vitepress/config.mts:94]: according to the docs I think you have to do it like this:

```
    'overview.md': 'software/index.md',
    'de/overview.md': 'de/software/index.md',
```

Comment (klml) [reply .vitepress/config.mts:94]: @FabianWilms you are right! thx

but did not work either

Comment (FabianWilms) [reply .vitepress/config.mts:94]: > but did not work either

what exactly did not work? After a build there should be an index.html-File under the `software` directory with the rendered contents of `overview.md`.

It won't automatically redirect from `opensource.muenchen.de/overview.html` to `opensource.muenchen.de/software/index.html`

File: .vitepress/config.mts [hunk 2]
```diff
@@ -5,6 +5,7 @@ const base = '/';
 
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
+
```
Comment (FabianWilms) [inline .vitepress/config.mts:8]:
```suggestion
```

unneeded change

**Review by FabianWilms — APPROVED** [review]

File: .vitepress/config.mts (hunk 1 above)
Comment (FabianWilms) [reply .vitepress/config.mts]: fixed

**Pull request merged**